src/waypoint.jsx

```diff
@@ -264,16 +269,15 @@ export default class Waypoint extends React.Component {
   }
 
   _getBounds() {
-    const waypointTop = ReactDOM.findDOMNode(this).getBoundingClientRect().top;
+    const waypointTop = this._ref.getBoundingClientRect().top;
     let contextHeight;
     let contextScrollTop;
     if (this.scrollableAncestor === window) {
       contextHeight = window.innerHeight;
       contextScrollTop = 0;
     } else {
       contextHeight = this.scrollableAncestor.offsetHeight;
-      contextScrollTop = ReactDOM
-        .findDOMNode(this.scrollableAncestor)
+      contextScrollTop = this.scrollableAncestor
```

Comment:
Ah, you're right. I was thinking that passing a component instance in as a `scrollableAncestor` would work (by leveraging `findDOMNode`), but I took another look at the source and saw that it would error earlier up in the code.

Ignore me! 🙈